dissect/target/plugins/filesystem/ntfs/mft.py

```diff
@@ -107,9 +107,16 @@ def check_compatible(self):
         ntfs_filesystems = [fs for fs in self.target.filesystems if fs.__fstype__ == "ntfs"]
         return len(ntfs_filesystems) > 0
 
-    @export(record=[FilesystemStdRecordCompacted, FilesystemFilenameRecordCompacted])
-    @arg("--compact", type=bool)
-    def mft(self, compact=False):
+    @export(
+        record=[
+            FilesystemStdCompactRecord,
+            FilesystemFilenameCompactRecord,
+            FilesystemFilenameRecord,
+            FilesystemStdRecord,
```

Comment:
```suggestion
            FilesystemStdRecord,
            FilesystemFilenameRecord,
            FilesystemStdCompactRecord,
            FilesystemFilenameCompactRecord,
```
Might as well have put up a trigger warning with that ordering :smile: